### ISSUE TITLE
English translation

### ISSUE THREAD

**Issue by shteryana** [etabakov/fivekmrun-app#110]
It would be great if users of the app can switch to a version in English

**Comment by ivonasto** [comment]
Hey, can I grab this one as well?

**Comment by etabakov** [comment]
Yes, this is up for grabs. Keep in mind that it might be challenging, though. Some of the data shown in the application is coming from the server in Bulgarian, and there is no localization there. The app will have to convert labels and event names to English somehow.

**Comment by ivonasto** [comment]
You are referring to the events page I assume? I will start with the app first and tackle the external info later, if its ok. 

Do you want to use the locale of the user's device or have a button to switch between BG & EN?

**Comment by etabakov** [comment]
Yeah, mainly the event page, but I think there are labels here and there coming from the server as well. You will find them out in the process.

I'd suggest using the device's locale as the default and then leaving the user to change it with a button and remember their choice. But I haven't thought about it too much, so I might be wrong.

**Comment by etabakov** [comment]
implemented with #158
